sip/execution_framework_interface/docker_api/client/test/test_docker_client.py

```diff
@@ -0,0 +1,149 @@
+# -*- coding: utf-8 -*-
+"""Test for docker client API"""
+
+import logging
+import time
+
+from ..docker_client import DockerClient
+
+logging.basicConfig(level='DEBUG')
+
+DC = DockerClient()
+
+
+def test_create_services():
+    """ Test function for create services"""
+
+    # Remove all services and volumes
+    DC.delete_all_services()
+    assert not DC.get_service_list()
+    time.sleep(2)
+    DC.delete_all_volumes()
+    assert not DC.get_volume_list()
+
+    # Passing in a test compose file
+    filename = ["compose-file/docker-compose.visibility.yml"]
```

Comment:
- ideally path names should not require unit tests to be run from a specific location (ie avoid assumed relative paths). A work around to this is to make them relative to the location of the test file, eg.:

```python
from os.path import abspath, dirname, join

filename = abspath(join(dirname(__file__), '..', 'compose-file',
                   'docker-compose.visibility.yml'))
```